### ISSUE TITLE
Font Family With Double Quotes Pasted In Editor Causes Invalid HTML

### ISSUE THREAD

**Issue by jeromebridge** [nhn/tui.editor#1821]
## Describe the bug
When copying and pasting text with a common Microsoft font: `Segoe UI`, the following content is produced in the Markdown format.

It appears the problem is with the way the `font-family` is quoted using `"` rather than single quotes or no quotes. This results in invalid HTML and causes the HTML to show up in the WYSIWYG editor after saving the text.

```
<span style="color: rgb(23, 43, 77); font-family: "Segoe UI", sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;">Eagle 8% SPDA, CA PNC State Approval, Eagle Product Discontinuation</span>
```

## To Reproduce
1. Save this file as an `html` file:
    [Editor-Font-Family.html.txt](https://github.com/nhn/tui.editor/files/7287991/Editor-Font-Family.html.txt)
2. Open the file in the Chrome browser
3. Copy the line on the page onto the clipboard
4. Paste that line into the Markdown Editor (with the color-syntax plugin)
5. You should see Markdown produced like the following:
    ```
    So I added a new project, copied and pasted this description - <span style="font-size:10.5pt;font-family:"Segoe UI",sans-serif;color:#172b4d;background:white">Eagle 8% SPDA, CA PNC State Approval, Eagle Product Discontinuation</span>
    ```

## Expected behavior
The font family should do one of the following:
1. Do not use quotes
2. Use single quotes
3. Or escape the double quotes if the are used like `&quote;`

## Screenshots
![image](https://user-images.githubusercontent.com/5193916/136074332-8af93b3c-0320-4370-876e-b07dd65e1116.png)

## Desktop (please complete the following information):
 - OS: Ubuntu 20
 - Browser Chrome
 - Version  91.0.4472.101

## Smartphone (please complete the following information):
N/A

## Additional context
N/A

**Comment by js87zz** [comment]
@jeromebridge 
Well, I don't think it's a bug. 
Doesn't it work if you write style attributes in single quotes as below? 

```
<span style='color: rgb(23, 43, 77); font-family: "Segoe UI", sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;'>Eagle 8% SPDA, CA PNC State Approval, Eagle Product Discontinuation</span>
```
![스크린샷 2021-10-06 오전 10 50 48](https://user-images.githubusercontent.com/37766175/136128101-1b4f4dae-83b3-4b9d-bff0-cecded81bff6.png)


It is difficult for the editor to parse all the wrong html text.

**Comment by js87zz** [comment]
@jeromebridge 
Sorry for late replying.
Now I understand clearly.
There's a problem with converting. I'll fix this issue.
Thank you.

**Comment by js87zz** [comment]
@jeromebridge 
The bug has been fixed in `v3.1.2`. Thanks!
